src/server/multi_svr.c

```diff
@@ -279,3 +279,30 @@ init_msi()
 
 	return 0;
 }
+
+/**
+ * @brief
+ * 	Used to get serverer_instance_id which is of the form server_instance_name:server_instance_port
+ * 
+ * @return char *
+ * @return NULL - failure
+ * @retval !NULL - pointer to server_instance_id
+ */
+char *
+get_svr_inst_id(void)
```

Comment:
since this is actually allocating memory, how about 'create_svr_inst_id'/'generate_svr_inst_id' ?